docs/report.md

```diff
@@ -39,6 +39,17 @@ The other workflow that is being ran when a tag is pushed to main, is the workfl
 
 ## Team work
 
+When requirements from the course were published, the group would either delegate the different issues and users stories to be created by different group members, or one or two individuals would write them out by themselves as quickly as possible. This allowed the group to quickly be aware of what the current issues were, and allowed members to assign themselves to these issues.
+
+Similarly, if a bug was found, or group members agreed on a new functionality of the program, someone was quickly assigned to make an issue. A single issue was usually worked on by anywhere between 1 and 3 group members according to the scale of the issue.
+
+The group utilized some pair programming at the start of the project, although it developed into more individually focused programming. If a member needed help on they would usually contact other members on some communication platform.
```

Comment:
I would probably replace "communication platform" with just discord to shorten that a bit. Otherwise I think this is very good